### PR TITLE
DO NOT MERGE - Palette Virtual Cluster Rename fix

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR addresses the name change for nested clusters to Palette Virtual Clusters.

In addition, the following broken URLs have also been fixed:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/karlcardenas/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/karlcardenas/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

https://docs-latest.spectrocloud.com/user-management/rbac | 404 | BROKEN | https://docs-latest.spectrocloud.com/user-management/ |  
-- | -- | -- | -- | --
https://docs-latest.spectrocloud.com/integrations/csi/ | 404 | BROKEN | https://docs-latest.spectrocloud.com/troubleshooting |  
https://docs-latest.spectrocloud.com/clusters/nested-clusters | 404 | BROKEN | https://docs-latest.spectrocloud.com/release-notes |   |  
https://docs-latest.spectrocloud.com/release-notes/(/clusters/public-cloud/aks | 404 | BROKEN | https://docs-latest.spectrocloud.com/release-notes |   |  
https://github.com/dexidp/dex/blob/master/Documentation/kubernetes.md | 404 | BROKEN | https://docs-latest.spectrocloud.com/integrations/dex |  
https://www.nginx.com/products/nginx/kubernetes-ingress-controller/ | 403 | BROKEN | https://docs-latest.spectrocloud.com/integrations/nginx |  
https://docs.portworx.com/reference/lighthouse/ | 404 | BROKEN | https://docs-latest.spectrocloud.com/integrations/portworx |  
https://github.com/rook/rook/blob/master/Documentation/ceph-quickstart.md | 404 | BROKEN | https://docs-latest.spectrocloud.com/integrations/rook-ceph |  
https://docs-latest.spectrocloud.com/user-management/palette-rbac/user-management/palette-rbac/project-scope-roles-permissions | 404 | BROKEN | https://docs-latest.spectrocloud.com/user-management/palette-rbac

</body>

</html>

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/967#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @karl-cardenas-coding. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:147]: If you would like more information on managing your resources, see the links below.

Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:147]:
```suggestion
If you would like more information on managing your resources, see the links below.
```

**Review by ritawatson — CHANGES_REQUESTED** [review]
Your changes all look good, and I'm suggesting one additional change.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:147]: Great catch @ritawatson , we should not write as if achieving something is difficult, hence the congratulations. Awesome catch 👏🏻 . 

I do think we should change "see the links below" to "please review the links below". That way, we get away from ableist language.

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:147]:
```suggestion
Your Palette Virtual Cluster is now deployed. If you would like more information on managing your resources, please review the links below.
```

**Review by ritawatson — CHANGES_REQUESTED** [review]
As you suggest, added 'please' and removed extra 'to'. In other company documentation, I haven't included 'please' or 'thank you' in content or messages. Is that a style practice at Spectro Cloud that might be up for discussion?

**Comment by karl-cardenas-coding** [comment]
@ritawatson correct, we shouldn't use please/thank you.

File: content/docs/04.5-devx/04-resource-quota.md
Comment (resmaa) [inline content/docs/04.5-devx/04-resource-quota.md:36]: PVC is a common term in K8s domain which is for "persistent Volume Claim" , so may be we should not be abbreviating 'Palette Virtual Clusters' as PVC.

image

File: content/docs/04.5-devx/04-resource-quota.md
Comment (resmaa) [inline content/docs/04.5-devx/04-resource-quota.md:43]: PVC is a common term in K8s domain which is for "persistent Volume Claim" , so may be we should not be abbreviating 'Palette Virtual Clusters' as PVC.

**Review by resmaa — CHANGES_REQUESTED** [review]
Suggestion: PVC abbreviation to be revisited.

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/967#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/967#pullrequestreview-1172773192" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by resmaa — APPROVED** [review]
aprvd

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/967#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/967#pullrequestreview-1174437667" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/967#pullrequestreview-1186460411" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile.md:19]:
```suggestion
App Profiles are templates created with preconfigured services required for Palette Virtual Cluster deployment. App Profiles provide a way to drive consistency across virtual clusters. You can create as many app profiles as required. An App Profile can be created to meet specific types of workloads on Palette Virtual Clusters. For example, you may create a development app profile with Helm Charts, Manifest, different database applications, etc.
```
Sandbox cluster was used instead of virtual cluster

File: content/docs/04.5-devx/08-registries.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/08-registries.md:19]:
```suggestion
The Pack registry is a server-side application that stores and serves packs to its clients. Packs from a pack registry are retrieved and presented as options during the creation of a cluster profile. Palette supports the configuration of multiple registries.
```
Minor cleanup

File: content/docs/04.5-devx/08-registries.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/08-registries.md:22]:
```suggestion
The default pack registry is Spectro Cloud's public pack registry. It consists of several packs that make it simple for a user to quickly create a cluster profile and launch a Kubernetes cluster with their choice of integrations. Spectro Cloud maintains all packs in the default pack registry, this includes taking care of upgrades in the pack registry whenever required.
```
Avoid using the word "easy", use "simple" instead. Easy refers to something that does not require a great effort or that does not present lot of difficulties. Simple refers to something that is direct, straightforward not complex.

File: content/docs/04.5-devx/08-registries.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/08-registries.md:25]: Let's link to the CLI tool here. Whenever referencing an internal resource or tool, share the link to it.

File: content/docs/04.5-devx/08-registries.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/08-registries.md:27]:
```suggestion
Palette Dev Engine supports the following types of custom registries:
```
We can avoid having to update in the future by not specifying the number. Simply list the registries below. I also cleaned up the sentence as it read a bit awkwardly.

File: content/docs/04.5-devx/07-palette-virtual-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-palette-virtual-clusters/00-pause-restore-virtual-clusters.md:40]: @resmaa this section is taking a lot of assumptions, such as assuming the user is already at the cluster page or in the proper context.  Let's avoid making assumptions. 
The validation section should start with a short intro sentence and then instruct the user on how to validate the state of the cluster.

Observe the following example.

 ## Validation

You can verify the state of a cluster by reviewing the cluster details. To review the state of a cluster and its details, do the following steps.

First, navigate to the "Mani Menu" and click on **Virtual Clusters** . Next, click on the specific cluster you want to check the status. This will take you to the cluster detail page. On this page, look for a section titled **Status**. The status section displays the current state of the cluster.

**Review by karl-cardenas-coding — COMMENTED** [review]
@resmaa Please don't commit directly to the PR, use PR that targets this branch so that we can review the changes.  I found several items that need more attention, so let's avoid pushing directly to this branch.

File: content/docs/04.5-devx/07-palette-virtual-clusters/00-pause-restore-virtual-clusters.md
Comment (resmaa) [reply content/docs/04.5-devx/07-palette-virtual-clusters/00-pause-restore-virtual-clusters.md:40]: sure, done

File: content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md:45]: @resmaa, we need a validation step as this is a how to.

File: content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md:44]: Click on **Confirm**

File: content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md:29]: To Clone an App Profile, complete the following steps.


^ More of an active voice vs passive

File: content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md:4]: The description is used by Google and affects our SEO score. Let's make sure we add more context and explain what this page is for.

File: content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md:27]: This is not following the How-to layout. Combine Use Case into the introduction.

File: content/docs/00-release-notes.md
Comment (ritawatson) [inline content/docs/00-release-notes.md:78]:
```suggestion
* While deploying multiple apps in a Palette Virtual Cluster, if the deployment of one of the apps is blocked due to errors, then subsequent apps deployed to the same virtual cluster might also be stuck in deploying state.  Apply the following workarounds if you encounter the issue.
```

File: content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md
Comment (ritawatson) [inline content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md:20]:
```suggestion
There are certain advantages to using a virtual cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
```

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/967#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/967#pullrequestreview-1174437667" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/05-app-profile.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile.md:43]: Instead of reffering containers as methods can we refer its definition from here https://www.docker.com/resources/what-container/

File: content/docs/04.5-devx/05-app-profile.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile.md:46]: We should be referring only helm registries here and not the pack registry

File: content/docs/04.5-devx/05-app-profile.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile.md:55]: I think it would be nice to have it next to Manifest

File: content/docs/04.5-devx/05-app-profile.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile.md:46]: I think the link should be devx/registries/helm-registry

File: content/docs/04.5-devx/05-app-profile.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile.md:52]: We should have a section page for Database services with the list of macros that are available for each database service

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md
Comment (shubhamrajvanshi) [inline content/docs/04.5-devx/05-app-profile/02-container-deployment.md:61]: Change 
```suggestion
4. **Review** the information and complete the app profile creation wizard. Add more tiers to the profile per user requirements, and Palette Clusters can be deployed using the Deploy In A New Palette Virtual Cluster. 
```

**Review by shubhamrajvanshi — COMMENTED** [review]
Requested some changes

File: content/docs/04.5-devx/08-registries/00-helm-registry.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/08-registries/00-helm-registry.md:23]: In Palette users can either configure their own repos and use the charts out of the preconfigured out of box helm repos like Bitnami.

File: content/docs/04.5-devx/05-app-profile/02-container-deployment.md
Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/05-app-profile/02-container-deployment.md:61]: @resma, tomorrow when you fix the container deployment guide. Let's clean up this paragraph. The "and Palette Clusters can be deployed using the deployed app profile" is awkward and does not make sense.

**Pull request merged**
